Sources/XMTPiOS/Client.swift

```diff
@@ -462,6 +462,21 @@ public final class Client {
 		let fm = FileManager.default
 		try fm.removeItem(atPath: dbPath)
 	}
+	
+	@available(*, deprecated, message: "This function is delicate and should be used with caution. App will error if database not properly reconnected. See: reconnectLocalDatabase()")
```

Comment:
@insipx was able to mark this function as delicate to warn developers before using.